packages/forms/src/index.tsx

```diff
@@ -154,7 +204,18 @@ const coerceValues = (data, coerce) => {
   return coercedData
 }
 
-const FormWithCoercionContext = (props) => {
+interface FormWithCoercionContext
+  extends Omit<React.HTMLProps<HTMLFormElement>, 'onSubmit'> {
+  error?: any // TODO (error shape)
+  formMethods?: UseFormMethods
+  validation?: UseFormOptions // TODO: This naming seems weird
```

Comment:
We have this `validation` prop here. Further down, L227, we pass it in to `useForm`. So we've named it `validation`, but really it can be any/all options that are valid for `useForm`. 

@cannikin You wrote this code. What was the intention here? Should we only allow `mode`? (As we set in the tutorial, at the end, here https://redwoodjs.com/tutorial/everyone-s-favorite-thing-to-build-forms#validating-input-format) or should we keep it so that we can set all options for `useForm`? If so, I think we should rename the prop to reflect that.

Reply:
I think it makes sense to rename this so that it indicates that it's used for all options.

We can mark it as deprecated and remove it in the `v0.22.0` and introduce a new prop that does the same thing but is named something more appropriate.

Reply:
Great. Thanks. I'll keep it as it is right now then. 
And then another PR can add another prop and deprecate this one

Reply:
Oh god that was a long time ago. If it can accept all options then we should definitely rename!

Reply:
I created an issue for it earlier https://github.com/redwoodjs/redwood/issues/1434